dev/docker/docker-compose.dev.yml

```diff
@@ -167,6 +172,28 @@ services:
       - client
       - backend
       - autoupdate
+      - keycloak
+    environment:
+      - ACTION_HOST=backend
+      - ACTION_PORT=9002
+      - PRESENTER_HOST=backend
+      - PRESENTER_PORT=9003
+      - AUTOUPDATE_HOST=autoupdate
+      - AUTOUPDATE_PORT=9012
+      - ICC_HOST=icc
+      - ICC_PORT=9007
+      - SEARCH_HOST=search
+      - SEARCH_PORT=9050
+      - MEDIA_HOST=media
+      - MEDIA_PORT=9006
+      - MANAGE_HOST=manage
+      - MANAGE_PORT=9008
+      - CLIENT_HOST=client
+      - CLIENT_PORT=9001
+      - VOTE_HOST=vote
+      - VOTE_PORT=9013
+      - KEYCLOAK_HOST=keycloak
+      - KEYCLOAK_PORT=8080
```

Comment:
Here probably the `env_file: services.env` directive should be used as done for the other services instead of redefining the whole environment, doubling the code.